### PR TITLE
lily comments

### PR DESCRIPTION
**Pull request by jukent**
Addresses #337 chapters 1-10

### CODE REVIEW

File: foundations/quickstart.ipynb [hunk 1]
```diff
@@ -342,9 +342,15 @@
    "id": "c0f192ef",
    "metadata": {},
    "source": [
-    "This list contains a mix of `int`, `float`, and `str` (character string)."
+    "This list contains a mix of `int` (integer), `float` (floating point number), and `str` (character string)."
    ]
   },
+  {
```
Comment (dopplershift) [inline foundations/quickstart.ipynb:348]: Looks like this is creating an extra empty cell? Was that intentional?

File: foundations/why-python.md [hunk 2]
```diff
@@ -10,18 +10,18 @@ Want to know what these terms mean for you and your work? Read on!
...
 
 ## Object Oriented languages
 
-Objects are custom datatypes. For every custom datatype, you usually have a set of operations you might want to conduct. For example, if you have an object that is a list of numbers you might want to apply a mathematical operation, such as sum, onto this list object in bulk. Not every function can be applied to every datatype; it wouldn't make sense to apply a logarithm to a string of letters or to capitalize a list of numbers. Data and the operations applied to them are grouped together into one object.
+Objects are custom datatypes. For every custom datatype, you usually have a set of operations you might want to conduct. For example, if you have an object that is a list of numbers, and you might want to apply a mathematical operation, such as sum, onto this list object in bulk. Not every function can be applied to every datatype; it wouldn't make sense to apply a logarithm to a string of letters or to capitalize a list of numbers. Data and the operations applied to them are grouped together into one object.
```
Comment (dopplershift) [inline foundations/why-python.md:21]: I think the comma is good, but I don't think the "and" is better (or grammatically correct).

**Review by dopplershift — COMMENTED** [review]
A couple of minor comments, but otherwise this looks good.

File: foundations/quickstart.ipynb [hunk 3]
```diff
@@ -342,9 +342,15 @@
...
+   "cell_type": "markdown",
+   "id": "15d77b95",
+   "metadata": {},
+   "source": []
+  },
```
Comment (dopplershift) [inline foundations/quickstart.ipynb:353]: Looks like this is creating an extra empty cell? Was that intentional?

Comment (jukent) [reply foundations/quickstart.ipynb:353]: Oops good catch.

File: foundations/why-python.md [hunk 4]
```diff
@@ -10,18 +10,18 @@ Want to know what these terms mean for you and your work? Read on!
...
+Python as a language is open source, which means that there is a community of developers behind its codebase. Anyone can join the developer community and contribute to deciding the future of the language. When someone identifies gaps in Python's abilities, they can write up the code to fill these gaps. The open source nature of Python means that Python as a language is very adaptable to the shifting needs of the user community. This harks back to the idea that the wide-spread use of Python within the scientific community is a benefit to you! The large Python user base within your field has established high level community Python packages that are available to you in your workflow.
```
Comment (ktyle) [inline foundations/why-python.md:25]: harks --> harkens
wide-spread --> widespread

File: preamble/how-to-use.md [hunk 5]
```diff
@@ -115,8 +115,8 @@ the Jupyter Notebook server. For example if you downloaded the
 notebook file to your home directory you would do:
 
 ```
-% cd ~
-% jupyter notebook
+cd ~
+jupyter notebook
```
Comment (ktyle) [inline preamble/how-to-use.md:119]: I don't think we should deal with this in this specific PR, but more generally, shouldn't we be advising new users to run jupyter **lab**, not the deprecated jupyter **notebook**?

**Review by ktyle — COMMENTED** [review]
Looks good; I left one comment that should be addressed in this PR, and another comment whose resolution is (IMHO) out of scope for the intent of the PR.

File: preamble/how-to-use.md (hunk 5 above)
Comment (jukent) [reply preamble/how-to-use.md]: Good call. I am changing this to `jupyterlab` but we might want to add or look closer at the text to see if everything is still consistent.

**Review by dopplershift — APPROVED** [review]

**Review by ktyle — APPROVED** [review]
looks good!

**Pull request merged**